src/main/java/org/apache/datasketches/memory/Prim.java

```diff
@@ -52,7 +52,7 @@
  *
  * @author Lee Rhodes
  */
-enum Prim {
+public enum Prim {
```

Comment:
Is `Prim` an abbreviation for `Primitive`? Does it encapsulate offsets for different primitive types?